Misc/NEWS.d/next/macOS/2019-07-13-15-58-18.bpo-18049.MklhQQ.rst

```diff
@@ -0,0 +1,3 @@
+Increase the default stack size of threads from 5MB to 16MB on macOS, to
```

Comment:
nitpick:

```suggestion
Increase the default stack size of threads from 5 MiB to 16 MiB on macOS, to
```

Reply:
Is MiB something we use elsewhere? 

I'm probably old, but don't particularly like the MiB vs. MB distinction as it is often clear from the context what's meant.